<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>256</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Sans</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="appLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>701</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>DTS Youtube Downloader</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="downloadButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>60</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="urlBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>411</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="urlText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="text">
     <string>Youtube Video URL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="about">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>200</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>About...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reportButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Report...</string>
    </property>
   </widget>
   <widget class="QLabel" name="infoText">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>641</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;When you copy the URL of the video you want to download from YouTube, paste it into the box above, and then press the 'Download' button, your downloading process will begin. You can check the status above.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
